src/main/java/streamingapi/client/LoggerFactory.java

```diff
@@ -0,0 +1,24 @@
+package streamingapi.client;
```

Comment:
should this class be in this package ? Is it fully specific to streaming api client?